lib/widgets/chart/chart.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+
+import 'package:expense_tracker/widgets/chart/chart_bar.dart';
+import 'package:expense_tracker/models/expense.dart';
+
+/// A widget that displays a chart of expenses grouped by category.
+///
```

Comment:
Good that you added some comments even though this file was supplied by the course, it shows you understand the code